drivers/bridge/bridge.go

```diff
@@ -1009,6 +1012,10 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 		}
 	}
 
+	if err := d.storeUpdate(endpoint); err != nil {
+		logrus.Warnf("Failed to save bridge endpoint %s to store: %v", ep.id[0:7], err)
```

Comment:
The roll-back function should not be invoked given the if scope of `err`.
I do not want to fail the container start if we cannot save it to the bridge store.
The container may not even live long enough to go through a daemon reload.